elki/src/main/java/elki/math/statistics/dependence/MCDEDependenceMeasure.java

```diff
@@ -0,0 +1,249 @@
+/*
```

Comment:
This class should be moved to the `elki-core-math` module. The HICSDependenceMeasure is a special case, because it imports from the `HiCS` class, which is in the `elki-outlier` module, and we don't want a dependency from `elki-core-math` onto `elki-outlier` (but I'll try to resolve this by refactoring - most things in the `elki` main module should eventually be moved into a more appropriate place).